AlphaWallet/Core/CoinTicker/CoinTickersFetcher.swift

```diff
@@ -289,7 +289,11 @@ fileprivate struct Ticker: Codable {
                 return false
             }
         } else {
-            return symbol.localizedLowercase == tokenObject.symbol.localizedLowercase && name.localizedLowercase == tokenObject.name.localizedLowercase
+            if tokenObject.server == .xDai {
+                return symbol.localizedLowercase == "xDai".localizedLowercase && name.localizedLowercase == "xDai".localizedLowercase
```

Comment:
Do you think it will be better if we change the body of the `if` to also match "gno"/"GNO"? To support the future where CoinGecko changes the xDai/GNO `symbol` from "xDai" to "GNO"/"gno".